src/h5web/guards.ts

```diff
@@ -130,16 +129,6 @@ export function hasNumericType<S extends Shape>(
   return isNumericType(dataset.type);
 }
 
-function isIntegerType(type: DType): type is IntegerType {
-  return [DTypeClass.Integer, DTypeClass.Unsigned].includes(type.class);
-}
-
-export function hasIntegerType<S extends Shape>(
-  dataset: Dataset<S>
-): dataset is Dataset<S, IntegerType> {
-  return isIntegerType(dataset.type);
-}
-
```

Comment:
All the `IntegerType` guards are no longer needed as `RgbVis` supports `NumericType` now